controllers/terminal_controller.go

```diff
@@ -724,16 +732,34 @@ func deleteClusterRoleBinding(ctx context.Context, cs *ClientSet, name string) e
 	return deleteObj(ctx, cs, clusterRoleBinding)
 }
 
-func (r *TerminalReconciler) createOrUpdateAdminKubeconfig(ctx context.Context, targetClientSet *ClientSet, hostClientSet *ClientSet, t *extensionsv1alpha1.Terminal, labelsSet *labels.Set, annotationsSet *utils.Set) (*corev1.Secret, error) {
-	accessSecret, err := r.createAccessToken(ctx, targetClientSet, t, labelsSet, annotationsSet)
+type adminKubeconfig struct {
```

Comment:
The name `adminKubeconfig` is very missleading I think since it contains two secrets

Reply:
@holgerkoser maybe rename it to `type adminKubeconfigSecretBundle struct {` ?

Reply:
✅